redwood-treehouse-lazylayout-composeui/src/commonMain/kotlin/app/cash/redwood/treehouse/lazylayout/composeui/ComposeUiLazyList.kt

```diff
@@ -32,33 +36,51 @@ import app.cash.redwood.treehouse.TreehouseApp
 import app.cash.redwood.treehouse.TreehouseView.WidgetSystem
 import app.cash.redwood.treehouse.composeui.TreehouseContent
 import app.cash.redwood.treehouse.lazylayout.api.LazyListInterval
-import app.cash.redwood.treehouse.lazylayout.widget.LazyColumn
+import app.cash.redwood.treehouse.lazylayout.widget.LazyList
 
-internal class ComposeUiLazyColumn<A : AppService>(
+internal class ComposeUiLazyList<A : AppService>(
   treehouseApp: TreehouseApp<A>,
   widgetSystem: WidgetSystem,
-) : LazyColumn<@Composable () -> Unit> {
+) : LazyList<@Composable () -> Unit> {
+  private var isVertical by mutableStateOf(false)
   private var intervals by mutableStateOf<List<LazyListInterval>>(emptyList())
 
   override var layoutModifiers: LayoutModifier = LayoutModifier
 
+  override fun isVertical(isVertical: Boolean) {
+    this.isVertical = isVertical
+  }
+
   override fun intervals(intervals: List<LazyListInterval>) {
     this.intervals = intervals
   }
 
   override val value = @Composable {
-    LazyColumn(
-      horizontalAlignment = Alignment.CenterHorizontally,
-      modifier = Modifier
-        .fillMaxWidth(),
-    ) {
+    // TODO Remove statically sized containers (https://github.com/cashapp/redwood/pull/854).
+    val itemBoxModifier = if (isVertical) Modifier.height(64.dp) else Modifier.width(64.dp)
+    val content: LazyListScope.() -> Unit = {
```

Comment:
Hmm I assume because this isn't a composable lambda that we aren't able to use `movableContentOf`/`movableContentWithReceiverOf`. Probably fine since no one is changing the orientation at any rate where it would matter.